src/sections/About.html

```diff
@@ -0,0 +1,4 @@
+<section class="m-8 p-5 lg:text-[3rem] lg:m-8 lg:p-12 text-[1rem] md:p-8 md:text-[2rem]  font-outfit">
+    <p class="m">We are a Berlin-based communications agency specialising in helping companies and brands find their voice and position themselves. Whether it's brand building, identity sharpening, or communication strategy, we bring a fresh perspective to established structures, develop tailor-made communication solutions, and ensure that messages resonate. With over a decade of experience, we have honed our skills in giving brands their unmistakable identity and making them visible in the long term.</p>
+<img src="/Images/Yellow Destop image.png" class=" h-1/2 pt-20 pl-[20%] md:pl-[40%] lg:pl-[40%]">
```

Comment:
Best practice reminder to avoid arbitrary values as much as you can.

Reply:
Yeap  worked on both removing unnecessary arbitary values and removing the image